### PR TITLE
Create title.scss and colors.scss

### PR DESCRIPTION
**Pull request by tebanwolox**
## Summary
- Create colors.scss
- Create titles.scss
- Create assets folder

## Screenshot
Screen Shot 2019-05-13 at 18 04 16

## Trello Card
https://trello.com/c/MVlFGHt2/21-page-titles

### CODE REVIEW

File: scss/titles.scss [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  background-repeat: no-repeat;
+  display: flex;
+  height: 100px;
+  justify-content: flex-end;
+  max-height: 100%; 
+  max-width: 100%;
+  width: 225px;
+  
+  .title {
+      align-self: center;
```
Comment (nmartinelli23) [inline scss/titles.scss]: Correct indentation.

File: scss/titles.scss [hunk 2]
```diff
@@ -0,0 +1,17 @@
+.title-container { 
+  background-image: url("assets/download.svg");
+  background-repeat: no-repeat;
+  display: flex;
+  height: 100px;
+  justify-content: flex-end;
+  max-height: 100%; 
+  max-width: 100%;
+  width: 225px;
```
Comment (nmartinelli23) [inline scss/titles.scss]: Put the limit value in max-width and width in 100% (same in height).

File: scss/titles.scss [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+  
+  .title {
+      align-self: center;
+      color: $black-color;
+      font-size: 1.5em;
```
Comment (nmartinelli23) [inline scss/titles.scss]: Use px unit please.

File: index.html [hunk 4]
```diff
@@ -8,6 +8,8 @@
   <link rel="stylesheet"  type="text/css" href="application.css">
 </head>
 <body>
-  <h1 class="title">Hello World</h1> 
+  <div class="title-container">
```
Comment (henryzarza) [inline index.html]: This div is unnecessary. You can do it without this

File: scss/titles.scss [hunk 5]
```diff
@@ -0,0 +1,17 @@
+.title-container {
```
Comment (henryzarza) [inline scss/titles.scss]: You don't need this, try with `pseudo elements`. Check this [documentation](https://developer.mozilla.org/en-US/docs/Web/CSS/::before)

File: scss/titles.scss [hunk 6]
```diff
@@ -0,0 +1,17 @@
+.title-container { 
+  background-image: url("assets/download.svg");
```
Comment (henryzarza) [inline scss/titles.scss]: Change this name for something more descriptive

**Pull request merged**